src/navigation/menu_bar.py

```diff
@@ -114,48 +114,50 @@ async def coroutine(self: MenuNav) -> None:
 
             If the path entered is a valid file name, save the current note at that path.
             """
+            dialog = ScrollMenuDialog(
+                title="Save As",
+                text="Choose the location of the file.",
+                directory=self.application_state.current_dir,
+                show_files=False,
+            )
+            directory = await self.show_dialog_as_float(dialog)
```

Comment:
wondering if we can show what the selected on the left side of the scroll which would be done in ScrollMenuDialog